pkg/psmdb/mongos.go

```diff
@@ -359,6 +359,13 @@ func volumes(cr *api.PerconaServerMongoDB, configSource VolumeSourceType) []core
 }
 
 func MongosService(cr *api.PerconaServerMongoDB) corev1.Service {
+	ls := map[string]string{
+		"app.kubernetes.io/name":       "percona-server-mongodb",
+		"app.kubernetes.io/instance":   cr.Name,
+		"app.kubernetes.io/managed-by": "percona-server-mongodb-operator",
+		"app.kubernetes.io/part-of":    "percona-server-mongodb",
+		"app.kubernetes.io/component":  "mongos",
+	}
```

Comment:
let's use `clusterLabels` function from the same package and append `app.kubernetes.io/component` to it

Reply:
fixed in 7762104af1cb4ee68f3afa70caa7852fb0548d82